pkg/trace/config/apply.go

```diff
@@ -382,6 +383,28 @@ func (c *AgentConfig) applyDatadogConfig() error {
 		c.LogThrottling = false
 	}
 
+	if config.Datadog.GetBool("apm_config.internal_profiling.enabled") {
+		profilingSite := config.Datadog.GetString("internal_profiling.profile_dd_url")
+		if profilingSite == "" {
+			profilingSite = site
+		}
+
+		c.ProfilingSettings = &profiling.Settings{
```

Comment:
These settings seems to be read in every agent. That's a lot of duplication. Can't we just have them in one place? Some utility which returns them for everyone?

Reply:
That's what this PR is for :)